README.md

```diff
@@ -8,29 +8,15 @@ not prevent clean-up of other resources.
 
 ### Sample Usage
 
-`Cleanups` accepts a list of `AutoCloseable` and `RunnableWithException`.
-Upon calling `doAll()` all `AutoCloseable` will be called and all `RunnableWithException` statements will be executed.
+`Cleanups` accepts a list of `AutoCloseable`(s). Upon calling `doAll()` all `AutoCloseable`(s) will be called.
```

Comment:
Replace:
AutoCloseables > AutoCloseables and Lambda callbacks

Although technically they are the same, but mentioning it helps the reader.